File: Python/tests/test_ug_bayes.py
import pytest

from hbayesdm.models import ug_bayes


def test_ug_bayes():
    _ = ug_bayes(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ug_bayes.R
context("Test ug_bayes")
library(hBayesDM)

test_that("Test ug_bayes", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ug_bayes(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
